completions/ip

```diff
@@ -289,7 +302,20 @@ _comp_cmd_ip()
                     # TODO
                     ;;
                 show | flush)
-                    # TODO
+                    case "$prev" in
+                        nud)
+                            _comp_cmd_ip__neigh_states "$1"
+                            ;;
+                        dev)
+                            _comp_compgen_available_interfaces "$1"
+                            ;;
+                        proxy | to | vrf) # TODO - Maybe we can complete vrf here?
+                            :
+                            ;;
+                        *)
+                            _comp_compgen -- -W 'proxy to nud vrf dev'
```

Comment:
Should we handle `nomaster` here, and also in switch case conditions above?

Reply:
I think we are not doing it for any other similar cases in `ip` either, so hardcoding it would be fine with me.

Reply:
OK, done. forced push

Reply:
LGTM, one conflict needs to be resolved, good to go otherwise.

Reply:
O fixed the conflict